make/compiler/Makefile.clang

```diff
@@ -123,10 +123,11 @@ endif
 #
 # Flags for turning on warnings for C++/C code
 #
-WARN_CXXFLAGS = -Wall -Werror -Wpointer-arith -Wwrite-strings -Wno-strict-aliasing -Wmissing-declarations
+WARN_CXXFLAGS = -Wall -Werror -Wpointer-arith -Wwrite-strings -Wno-strict-aliasing -Wmissing-declarations -Werror=zero-as-null-pointer-constant
 WARN_CFLAGS = $(WARN_CXXFLAGS) -Wmissing-prototypes -Wstrict-prototypes -Wmissing-format-attribute
 WARN_GEN_CFLAGS = $(WARN_CFLAGS)
 SQUASH_WARN_GEN_CFLAGS = -Wno-unused -Wno-uninitialized
+NO_NULLPTR_ERRROR = -Wno-error=zero-as-null-pointer-constant
```

Comment:
ERRROR -> ERROR  (and in Makefile.gnu) as well

Reply:
My terminal must have been messed up (often a bad terminal state causes characters to be repeated without showing them on the screen).